src/main/java/core/basesyntax/SqrtOperation.java

```diff
@@ -7,7 +7,11 @@
  * где вместо вопросика будет подставлено фактически переданное в метод число.</p>
  */
 public class SqrtOperation {
-    public double sqrt(double number) {
-        return 0.0;
+    public double sqrt(double sqrtNumber) {
+        if (sqrtNumber < 0) {
+            throw new IllegalArgumentException("Expected non-negative number, got " + sqrtNumber);
+        } else {
```

Comment:
why you are using else? do you know how if statement works? what happens when you are using return or throwing the exception inside the if?